content/company/news.mdx

```diff
@@ -11,6 +11,24 @@ subTitle: >
   page](/company/history).
 
 
+  ## MAY 2024 - SSW/SSW TV sponsors Serverless Days ANZ
+
+
+  SSW sponsors [Serverless Days ANZ](https://anz.serverlessdays.io/) in both Sydney and Auckland.
+
+
+## MAY 2024 - SSW/SSW TV sponsors /NEW 2024
```

Comment:
This should fix the build error maybe just make this PR a draft and check the changes again 

```suggestion
  ## MAY 2024 - SSW/SSW TV sponsors /NEW 2024
```